velero-plugins/common/types.go

```diff
@@ -1,37 +1,31 @@
 package common
-
 type routingConfig struct {
-	Subdomain string `json:"subdomain"`
```

Comment:
Think we need to update this file with tabs

Reply:
Ok, will try to update it once again. It already is with tabs based on the setting but will commit it again.